application/apache-kafka-flink-streaming/README.md

```diff
@@ -0,0 +1,135 @@
+# Streaming data with Apache Kafka, Apache Flink and CrateDB.
+
+## About
+
+This example showcases what a data-streaming architecture leveraging Kafka and Flink could look
+like.
+
+We use.
+
+- Kafka (confluent)
+- Apache Flink
+- CrateDB 
+- Python >=3.7<=3.11
+- 
+[img of the thing]
```

Comment:
Either add an image, or remove this line? When I am adding screenshots for those purposes, I am usually not adding them to the content tree in the repository itself. Instead, I am pasting them into a an issue comment on GitHub I will never submit, and then grab the image URL out of the text field after the image upload concluded.